docs/src/developers/style.md

```diff
@@ -134,8 +146,14 @@ val queueOut = Queue(
 ## Naming Conventions
 
 Chisel follows the [Scala Naming Conventions](http://docs.scala-lang.org/style/naming-conventions.html).
-In general, Chisel code should use CamelCase for naming (ie. the first letter
-of each word is capitalized except sometimes the first word).
+In general, Chisel code should use `camelCase` for variable naming (ie. the first letter
+of each word is capitalized except sometimes the first word) and `HeadlessCamelCase` for class names.
```

Comment:
```suggestion
In general, Chisel code should use `lowerCamelCase` for variable naming (ie. the first letter
of each word is capitalized except for the first word) and `UpperCamelCase` for class names.
```
While I agree with @hcook that "HeadLessCamelCase" is funny, the standard terms are lowerCamelCase and UpperCamelCase.

We could also say something about how camelCase is useful because "It helps distinguish between user-provided names and compiler-inserted `_` in the emitted Verilog" or something like that.

Reply:
haha sorry I was legitimately taught `HeadlessCamelCase`, will change it to the other term (which is used below).

@jackkoenig your point is made in the more detailed section below